src/theory/strings/regexp_operation.cpp

```diff
@@ -481,8 +505,8 @@ int RegExpOpr::derivativeS( Node r, CVC4::String c, Node &retNode ) {
         break;
       }
       default: {
-        //Trace("strings-error") << "Unsupported term: " << mkString( r ) << " in derivative of RegExp." << std::endl;
-        Unreachable();
+        Assert(!isRegExpKind(r.getKind()));
+        break;
```

Comment:
I think that we also have to add a `ret = 0` here to say that the result is unknown (`ret` is 1 by default).